packages/svelte-ux/src/lib/utils/array.ts

```diff
@@ -188,7 +188,7 @@ export function nestedFindByPredicate(
   return undefined;
 }
 
-export type TreeNode = { id: string; name: string; level: number; children: Node[] };
+export type TreeNode = { id: string; name: string; level: number; children: TreeNode[] };
```

Comment:
I think this was meant to be `TreeNode`, or that the proper TS type for `Node` wasn't imported, as it currently appears to be using the browser-default `Node` type.